README.md

```diff
@@ -168,6 +169,20 @@ with the command `pca-wizard-templates  --emails`.
 | interface_type | Type of interface GoPhish will use with mail server. | string | "SMTP" | no |
 | ignore_cert | Indicate if GoPhish should ignore certs with mail server. | boolean | `True` | no |
 
+## User Report Field Dictionary ##
+
+User report JSONs are also exported by ```gophish-export```. User reports
+summarize data from targeted user clicks on phishing emails generated by a
+campaign. These JSON files are later imported into the PCA database.
+
+| Name | Description | Type | Default | Required |
+|------|-------------|:----:|:-------:|:--------:|
+| customer | A corresponding customer ID that this user report document is associated with. It is left null as it will be filled in later by a PCA operator or another tool prior to importation | string | null | No |
+| assessment | A corresponding assessment ID that this user report is associated with. | string | | yes |
+| campaign | A corresponding campaign ID that this user report is associated with. | string| | yes |
+| first_report | The first report (click) generated by a targeted user. Formatted by YYYY-MM-DDT: hh:mm.ss"| datetime | | yes |
+| total_num_reports | The total number of user reports received during a campaign.| integer | | no
```

Comment:
I thought over in [this discussion](https://github.com/cisagov/pca-core/pull/5#discussion_r651079451), we were saying that `total_num_reports` cannot be blank, i.e. it is required?

Reply:
See bac2cb88e00b183bdbae8f990c3ad5a48b01ef05

Reply:
It looks like you changed this field to be required in ccdf910, but then overwrote it with bac2cb8. I assume that you did not mean to do that. Please let me know when it's fixed.

Reply:
See cc24b7780b4ff24ca8f43633d2dce6a214d86892

Reply:
https://github.com/cisagov/gophish-tools/commit/cc24b7780b4ff24ca8f43633d2dce6a214d86892 fixes the alphabetization issue, but `total_num_reports` is still listed as optional.  Did you mean to put this comment in the conversation above?

Reply:
Nah, it was an oversight. See 51786dc6d5b6d82de55ecbf822644fe2fe43ce71